internal/collector/filter_test.go

```diff
@@ -179,3 +179,54 @@ func TestIncludedObjectAPIGroup(t *testing.T) {
 		})
 	}
 }
+
+func TestShouldMaskReason(t *testing.T) {
+	testCases := []struct {
+		desc        string
+		controllers []string
+		event       *v1.Event
+		expect      bool
+	}{
+		{
+			desc:        "IncludedSource",
+			controllers: []string{"default-scheduler", "kubelet"},
+			event:       &v1.Event{Source: v1.EventSource{Component: "kubelet"}},
+			expect:      false,
+		},
+		{
+			desc:        "ExcludedSource",
+			controllers: []string{"kubelet"},
+			event:       &v1.Event{Source: v1.EventSource{Component: "default-scheduler"}},
+			expect:      true,
+		},
+		{
+			desc:        "IncludedController",
+			controllers: []string{"kubelet", "default-scheduler"},
+			event:       &v1.Event{ReportingController: "kubelet"},
+			expect:      false,
+		},
+		{
+			desc:        "ExcludedController",
+			controllers: []string{"kubelet"},
```

Comment:
You are missing two or more controllers case like the one above for IncludedController for a true expected?